src/kilt/Kilt.ts

```diff
@@ -22,7 +24,16 @@ export async function disconnect(host: string): Promise<void> {
   clearCache()
 }
 
+export function config(configuration: {
+  host?: string
+  logLevel?: LogLevel
+}): void {
+  const { host, logLevel } = configuration
+  setConfiguration(host, logLevel)
+}
+
```

Comment:
what do you think of making this more versatile for the future by allowing arbitrary fields? We could store a `Map` of config options and retrieve them with a general `get()`. Getting the default host would thus look like `Kilt.config.get('host')`, setting it could be something like `Kilt.config.set({host: 'ws://....'})`
We could still maintain an interface that tells users which fields we currently use, but that way we can much easier extend it. You could set it up as a type generic that takes a type which defines known config options.

Reply:
additionally you could define something like `getOrError` which throws a general error (like `CONFIG_NOT_SET`) when users try to rely on a config option but forgot to set it.